recipe/meta.yaml

```diff
@@ -1,34 +1,33 @@
-{% set version = "5.9.3" %}
+{% set version = "5.13.2" %}
 
 package:
   name: isort
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/isort/isort-{{ version }}.tar.gz
-  sha256: 9c2ea1e62d871267b78307fe511c0838ba0da28698c5732d54e2790bf3ba9899
+  sha256: 48fdfcb9face5d58a4f6dde2e72a1fb8dcaf8ab26f95ab49fab84c2ddefb0109
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - isort = isort.main:main
 
 requirements:
   host:
-    - python >=3.6.1,<4.0
+    - python
     - pip
     - wheel
     - setuptools
     - poetry-core >=1.0.0
   run:
-    - python >=3.6.1,<4.0
+    - python
 
 test:
   imports:
     - isort
-    - isort._future
   commands:
     - pip check
     - isort --help
```

Comment:
Based on this: https://github.com/PyCQA/isort/blob/5.13.2/scripts/test.sh
You could try:
`pytest tests/unit/ -s --ignore=tests/unit/test_deprecated_finders.py`
With source_files `tests/unit/`
And required dependencies a subset of https://github.com/PyCQA/isort/blob/5.13.2/poetry.lock#L35